### ISSUE TITLE
A11y_AzureDataStudioExtensionAzureSQLMigration_AssessmentResultsAndRecommendations_ScreenReader: ORCA  is not announcing "Choose your azure sql target", Azure recommendation(preview), "to migrate to azure sql managed instance, view assessment results and select one or more database" label names along with data.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#23897]
### Actual Result:
ORCA  is not announcing "Choose your azure sql target", Azure recommendation(preview), "to migrate to azure sql managed instance, view assessment results and select one or more database" label names along with data.

**Similar issue is also observed in below scenarios:**
**Issue 1: Azure Database Migration Service_ Create new Azure Database migration service:**

1. Install Azure data studio application and open it
2. TAB till Extension press ENTER key and install Azure SQL Migration extension.
3. TAB till Respective server and right click. TAB till manage and press ENTER key.
4. TAB till Azure SQL Migration and press ENTER key. TAB till "migrate to azure sql" and press ENTER.
5. Select start new session and fill all the data and press ENTER on next button. 
6. Select the respective databases and ENTER on next button. 
7. Fill all the necessary details and press ENTER key on Next button till "Azure database migration service" page appears.
8. Start ORCA screen reader. TAB till "Create new " button and hit ENTER on it.
9. "Create azure database migration service" page appears. TAB to "Name" required field.
10. Observe that ORCA screen reader is not announcing "Name" edit field label name along with edit box.

### Expected Result:
ORCA screen reader should announce "Choose your azure sql target", Azure recommendation(preview), "to migrate to azure sql managed instance, view assessment results and select one or more database" label names along with data.

### Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #Benchmark; #ORCA; #WCAG1.3.1; #FTP; #SH-AzureDataStudioExtensionAzureSQLMigration-Linux-Jul23; #SQL Migration extension for Azure Data Studio; #Linux; #Element:Dropdown;

### Environment Details:
Application Name: Azure Data Studio extension 'Azure SQL Migration'
Operating System: Ubuntu 20.04.2 LTS
Azure data studio 1.44.1
Azure SQL Migration v1.4.8
Screen Reader: ORCA

### Repro Steps:

1. Install Azure data studio application and open it
2. TAB till Extension press ENTER key and install Azure SQL Migration extension.
3. TAB till Respective server and right click.
4. TAB till manage and press ENTER key.
5. TAB till Azure SQL Migration and press ENTER key.
6. TAB till "migrate to azure sql" and press ENTER.
7. Select start new session and fill all the data and press ENTER on next button. 
12. Select the respective databases and ENTER on next button. Start ORCA screen reader.
13. Assessment results and recommendations page will open. Observe the issue.

### User Impact:
The screen reader users will not get correct information about the control and will not get purpose of it.

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Attachment:

[23897_Issue.webm](https://github.com/microsoft/azuredatastudio/assets/90901519/487c5acc-9ad3-483b-810d-e14b163c3154)

**Similar issue 1 attachment:**

https://github.com/microsoft/azuredatastudio/assets/90901519/125692de-5854-4416-9443-939fa74cdaca

**Comment by krritik** [comment]
Hi @PoojaNamde,

For the issue, "**ORCA is not announcing "Choose your azure sql target", Azure recommendation(preview), "to migrate to azure SQL Managed instance, view assessment results and select one or more database" label names along with data.**

From the audio, it is not clear what ORCA announces at each of these steps. So, can you please see once and say the exact things announced by ORCA, so it helps in understanding the issue.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:09-11-23;

Verified this issue in the Azure data studio 1.46.0 version. Issue is fixed hence closing the bug.
